service/history/timerQueueStandbyProcessor.go

```diff
@@ -169,12 +197,13 @@ func (t *timerQueueStandbyProcessorImpl) processExpiredUserTimer(timerTask *pers
 				//
 				// we do not need to notity new timer to base, since if there is no new event being replicated
 				// checking again if the timer can be completed is meaningless
-				t.timerQueueAckMgr.retryTimerTask(timerTask)
-				return nil
+				return newTaskRetryError()
```

Comment:
instead of creating a new error each time lets just define this as a global variable and reuse the same error.

Reply:
ok